basicapp/spec.go

```diff
@@ -39,10 +39,11 @@ func (cc Chart) Validate() error {
 type ChartResources struct {
 	DaemonSets  []DaemonSet
 	Deployments []Deployment
+	Services    []Service
 }
 
 func (cr ChartResources) Validate() error {
-	if len(cr.DaemonSets) == 0 && len(cr.Deployments) == 0 {
+	if len(cr.DaemonSets) == 0 && len(cr.Deployments) == 0 && len(cr.Services) == 0 {
```

Comment:
I'm not sure we need to add this but if we do the error message needs to change.